src/AuthenticationService.php

```diff
@@ -30,6 +30,8 @@
 
 /**
  * Authentication Service
+ *
+ * @mixin \Authentication\Controller\Component\AuthenticationComponent
```

Comment:
I don't think this is correct. This class does not proxy methods to the AuthenticationComponent at all. What call is causing that phpstan error in your application?

Reply:
I'm sorry.
The error was due to my mistake.

The following is correct.

```
/**
 * @property AuthenticationComponent $Authentication
 */
class AppController extends Controller
{
    // ...

    public function initialize(): void
    {
        parent::initialize();

        $this->loadComponent('Authentication.Authentication');

        // ...
    }

    // ...
}
```

However, I had made the following mistakes.

```
/**
 * @property \Authentication\AuthenticationService $Authentication
 */
class AppController extends Controller
{
    // ...

    public function initialize(): void
    {
        parent::initialize();

        $this->loadComponent('Authentication.Authentication');

        // ...
    }

    // ...
}
```